### DISCUSSION TITLE
Help needed with uninstall launchctl

### DISCUSSION THREAD

**Discussion by viking1304** [Homebrew/discussions#2761] [Tap maintenance and Homebrew development] [answered]
I created new cask for Jump Desktop Connect, which is free server application that allows high performance desktop sharing (needs Jump Desktop client to connect). And made a PR

https://github.com/Homebrew/homebrew-cask/pull/117178

Unfortunately I had some struggle with uninstall because it require both pkgutil and launchctl. I managed to solve that but I am now facing a problem that I had no idea how to solve.

I am getting this error:

```
Run brew ruby -r "$(brew --repository homebrew/cask)/cmd/lib/check.rb" <<'EOF'
Error: Some launch jobs were not unloaded, add them to uninstall launchctl:
com.p5sys.jump.connect.49240.FBE2D2D2-10D9-4296-AB35-96FEAAE2A738
Error: Some launch jobs were not unloaded, add them to uninstall launchctl:
com.p5sys.jump.connect.49240.FBE2D2D2-10D9-4296-AB35-96FEAAE2A738
Error: Process completed with exit code 1.
```

The main problem is that this guid changes every time.

I tried to use "com.p5sys.jump.connect.49240.*" but that doesn't work.

I am stuck and I need help from someone with more experience.

**Comment by neutric** [comment, marked as answer]
This GUID is the Hardware UUID which one would expect to be unique for every machine (or CI run which seems to set up a fresh "virtual" machine). I'm unfortunately not versed with macOS service management, but [adobe-creative-cloud.rb](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/adobe-creative-cloud.rb#L35-L39) might be a starting point.

**Comment by viking1304** [comment]
Thank you @neutric. It helped to some extent, but I still would not be able to finish this without @miccal's help.

**Reply by neutric** [reply]
Glad you guys got it sorted out!
